src/env.cc

```diff
@@ -2260,4 +2271,97 @@ v8::CpuProfile* Environment::StopCpuProfile(v8::ProfilerId profile_id) {
   return profile;
 }
 
+Mutex Environment::notifications_mutex_;
+uint64_t Environment::next_notification_id_(0);
+std::unordered_map<uint64_t, Environment*> Environment::notifications_;
+
+void Environment::RegisterNotification(
+    const v8::FunctionCallbackInfo<v8::Value>& args) {
```

Comment:
Nit: add `v8::FunctionCallbackInfo` to the using list at the top of the namespace and drop the `v8::` prefix here.

Reply:
Thanks, I've done everywhere. Will be in the next push.